subsys/bluetooth/host/classic/avdtp.c

```diff
@@ -1105,11 +1104,10 @@ int bt_avdtp_l2cap_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
 	return 0;
 }
 
-static const struct bt_l2cap_chan_ops signal_chan_ops = {
-	.connected = bt_avdtp_l2cap_connected,
-	.disconnected = bt_avdtp_l2cap_disconnected,
-	.recv = bt_avdtp_l2cap_recv
-};
+static const struct bt_l2cap_chan_ops signal_chan_ops = {.connected = bt_avdtp_l2cap_connected,
+							 .disconnected =
+								 bt_avdtp_l2cap_disconnected,
+							 .recv = bt_avdtp_l2cap_recv};
```

Comment:
This doesn't help readability at all. Please revert this change (it will not cause CI failures, just warnings)

Reply:
A good solution for this is to always end struct initialization with a final `,`.

Firstly that makes clang-format format it much better
Secondly, this makes adding additional lines to the struct initialization a single line instead of `,\n<new_line>\n`